src/algorithms/pid/MergeParticleID.cc

```diff
@@ -0,0 +1,199 @@
+// Copyright 2023, Christopher Dilks
+// Subject to the terms in the LICENSE file found in the top-level directory.
```

Comment:
I think we are good to LGPL distribution of all new code instead of vague references to public domain
```suggestion
// SPDX-License-Identifier: LGPL-3.0-or-later
// Copyright (C) 2023, Christopher Dilks
```